packages/starknet-snap/src/getErc20TokenBalance.ts

```diff
@@ -53,23 +49,26 @@ export async function getErc20TokenBalance(params: ApiParams) {
       `getErc20Balance:\nerc20Address: ${erc20Address}\nuserAddress: ${userAddress}`,
     );
 
-    // For deployed accounts, use the PENDING block to show balance updates as soon as possible,
-    // facilitating concurrent transactions without delays.
-    // For non-deployed accounts, use the LATEST block to avoid displaying non-zero balances
-    // from pending transactions, because confirmed non-zero balance is required for deployment.
-    const blockIdentifier = (await isAccountDeployed(network, userAddress))
-      ? BlockIdentifierEnum.Pending
-      : BlockIdentifierEnum.Latest;
-    const balance = await getBalance(
+    const balanceLatest = await getBalance(
       userAddress,
       erc20Address,
       network,
-      blockIdentifier,
+      BlockIdentifierEnum.Latest,
+    );
+    const balancePending = await getBalance(
+      userAddress,
+      erc20Address,
+      network,
+      BlockIdentifierEnum.Pending,
     );
 
-    logger.log(`getErc20Balance:\nresp: ${toJson(balance)}`);
+    const resp = {
+      balancePending,
+      balanceLatest,
```

Comment:
how about change the response to MM standard? we may need to discuss

API Response reference - https://github.com/MetaMask/keyring-api/blob/main/src/api/keyring.ts

Caip 19 asset id - https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-19.md

e.g 
# DAI Token
starknet:SN_GOERLI/erc20:0x6b175474e89094c44da98b954eedeac495271d0f